### ISSUE TITLE
Can not write to a database other than the default

### ISSUE THREAD

**Issue by Redni** [strapi/strapi#332]
**Node.js version**: 8.8.1
**npm version**: 5.5.1
**Strapi version**: 3.0.0-alpha.6.4
**Operating system**: Arch Linux

Can not write to a database other than the default.

Database set in Settings Manager -> ENVIRONMENTS -> Database. Correct data, the database exists. Checked by terminal.

Then create a new collection in "Content Type Builder" and create a new field with a different Mongo base than the default. Click "Save" the following error:

```
INFO (8278 on inder): The server is restarting

ERROR (8318 on inder): Impossible to register the `test` model.
ERROR (8318 on inder): Cannot assign to read only property 'test' of object '#<Object>'
    TypeError: Cannot assign to read only property 'test' of object '#<Object>'
    at _.forEach (/home/inder/strapi/node_modules/strapi-mongoose/lib/index.js:146:38)
    at /home/inder/strapi/node_modules/lodash/lodash.js:4944:15
    at baseForOwn (/home/inder/strapi/node_modules/lodash/lodash.js:3001:24)
    at /home/inder/strapi/node_modules/lodash/lodash.js:4913:18
    at Function.forEach (/home/inder/strapi/node_modules/lodash/lodash.js:9359:14)
    at _.after (/home/inder/strapi/node_modules/strapi-mongoose/lib/index.js:83:15)
    at /home/inder/strapi/node_modules/lodash/lodash.js:10014:23
    at _.after (/home/inder/strapi/node_modules/strapi-mongoose/lib/index.js:205:15)
    at /home/inder/strapi/node_modules/lodash/lodash.js:10014:23
    at _.forEach (/home/inder/strapi/node_modules/strapi-mongoose/lib/index.js:294:15)
    at /home/inder/strapi/node_modules/lodash/lodash.js:4944:15
    at baseForOwn (/home/inder/strapi/node_modules/lodash/lodash.js:3001:24)
    at/home/inder/strapi/node_modules/lodash/lodash.js:4913:18
    at Function.forEach (/home/inder/strapi/node_modules/lodash/lodash.js:9359:14)
    at _.forEach (/home/inder/strapi/node_modules/strapi-mongoose/lib/index.js:210:15)
    at /home/inder/strapi/node_modules/lodash/lodash.js:4944:15
```

**Comment by lauriejim** [comment]
Okay, actually you can update the settings of the default connection but you can't have two mongo connections in the same time. But you can have one on mongo and one other for postgres. 

We will fix it soon.

**Comment by lauriejim** [comment]
It's fix on `strapi@3.0.0-alpha.7.2` - to update to this version, change the strapi version in your `package.json`. Install `strapi@3.0.0-alpha.7.2` in global and generate a new project. Then copy and past the new plugins and admin folder in your application.
